test/oryx/test_python_project.sh

```diff
@@ -28,5 +28,7 @@ check "oryx-build-python" oryx build --property python_version="${pythonVersion}
 check "oryx-build-python-installed" python3 -m pip list | grep mpmath
 check "oryx-build-python-result" python3 ./src/solve.py
 
+check "templates/8.0.1-does-not-exist" ls /usr/share/dotnet/templates/8.0.1 bash -c "test ! -d /usr/share/dotnet/templates/8.0.1"
```

Comment:
Hmm.. This isn't working as how I tested in my container. 
Never mind, let's remove this check.

Reply:
Now this is working. I have tested in my container and showing correct check. failing if the folder exists and passing if it is not exists.